frontend/src/metabase/css/core/colors.module.css

```diff
@@ -33,6 +33,12 @@
   --mb-color-text-primary: var(--mb-color-text-dark);
   --mb-color-text-secondary: var(--mb-color-text-medium);
   --mb-color-text-tertiary: var(--mb-color-text-light);
+  --mb-color-text-brand: var(--mb-base-color-brand-40);
+  --mb-color-background: var(--mb-base-color-white);
+  --mb-color-background-selected: var(--mb-base-color-brand-10);
+  --mb-color-background-disabled: var(--mb-color-bg-light);
+  --mb-color-background-inverse: var(--mb-color-bg-black);
+  --mb-color-background-brand: var(--mb-base-color-brand-40);
```

Comment:
@WiNloSt Could you add these semantic color mappings to `css-variables.ts` as well? It should mirror this `colors.module.css` file. If you do this, the SDK color mapping should be good to go.

Reply:
Fixed in https://github.com/metabase/metabase/pull/45813/commits/cca8d12d69e22df69ae584b392f0a14d7ccbe853 with a different solution. I also posted on [Slack here.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1722005859806329)

And I have confirmed that this issue doesn't present https://github.com/metabase/metabase/pull/45859